hue_h: 10
hue_l: 0
sat_h: 255
sat_l: 95
val_h: 255
val_l: 0
color_threshold: 15
